Fixed #31573 -- Made QuerySet.update() respect ordering on MariaDB/MySQL. (#13088)
django/django: docs/ref/models/querysets.txt

        e.comments_on = False
        e.save()

Ordered queryset
^^^^^^^^^^^^^^^^

.. versionadded:: 3.2

Chaining ``order_by()`` with ``update()`` is supported only on MariaDB and
MySQL, and is ignored for different databases. This is useful for updating a

> unique field in the order that is specified. For example::

Reviewer: ```suggestion
unique field in the order that is specified without conflicts. For example::
```

    Entry.objects.order_by('-number').update(number=F('number') + 1)

.. note::

>     The ``order_by()`` clause with annotations is always ignored.

Reviewer: ```suggestion
    If the ``order_by()`` clause contains annotations, it will be ignored.
```
